llama-index-integrations/vector_stores/llama-index-vector-stores-deeplake/llama_index/vector_stores/deeplake/base.py

```diff
@@ -137,6 +140,64 @@ def client(self) -> Any:
         """
         return self._vectorstore.dataset
 
+    def summary(self):
+        self._vectorstore.summary()
+
+    def get_nodes(
```

Comment:
Nice! 

If you wanted a follow-up PR, there is also `delete_nodes()` and `clear()` as well that were recently added to the base class

Reply:
Thanks! I'll get those added too